docs/content/manual/advanced/addons/5337-enable-addons-check-deployment.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Enable Harvester addons and check deployment state
+---
+
+* Related issues: [#5337](https://github.com/harvester/harvester/issues/5337) [BUG] Failed to enable vm-importer, pcidevices and harvester-seeder controller addons, keep stuck in "Enabling" state
+
+## Category: 
+* Addons
+
+## Verification Steps
+1. Prepare three nodes Harvester cluster
+1. Open Addvanced -> `Addons` page
```

Comment:
Advanced

Reply:
Thanks for the check. Update the typo in line.